test/reporters/tests/verbose.test.ts

```diff
@@ -157,8 +157,13 @@ test('does not render tree when in non-TTY', async () => {
      ✓ fixtures/verbose/example-1.test.ts > suite in root > suite in suite > test pass in nested suite #2 [...]ms
      × fixtures/verbose/example-1.test.ts > suite in root > suite in suite > suite in nested suite > test failure in 2x nested suite [...]ms
        → expected 'should fail' to be 'as expected' // Object.is equality
+     ↓ fixtures/verbose/example-1.test.ts > suite skip in root > test 1.3
```

Comment:
the old implementation bailed out if there were no test

the new reporter implementation always prints skipped tests, so this is actually a bug fix